test/e2e/framework/framework.go

```diff
@@ -84,6 +84,8 @@ type Framework struct {
 	NamespaceDeletionTimeout time.Duration
 	SkipPrivilegedPSPBinding bool // Whether to skip creating a binding to the privileged PSP in the test namespace
 
+	SecondaryNamespace *v1.Namespace
```

Comment:
Can we make `Namespace` into `Namespaces` instead?

Reply:
I have removed this bit entirely from the code and move namespace creation to storage utils.